static/js/app.js

```diff
@@ -3,106 +3,158 @@
+console.log(currentLabel);                                            //remove when done
+
+//testing purposes //remove when implementing dynamic adding 
+{
+  $('head').append(
+    $('<style/>', {
+      id: 'lorem-style',
+      html: '.hwt-content .label_lorem {background-color: rgb(255, 0, 0);}'
+    })
+  );
+  $('head').append(
+    $('<style/>', {
+      id: 'dolor-style',
+      html: '.hwt-content .label_dolor {background-color: rgb(255, 255, 0);}'
+    })
+  );
+  $('head').append(
+    $('<style/>', {
+      id: 'rem-style',
+      html: '.hwt-content .label_rem {background-color: rgb(255, 192, 203);}'
+    })
+  );
+  $('head').append(
+    $('<style/>', {
+      id: 'sit-style',
+      html: '.hwt-content .label_sit {background-color: rgb(173, 255, 47);}'
+    })
+  );
+}
 
 $(textArea).val(currentDocument.text);
 
 //When the user releases the mouse, highlight the selected text
 textArea.addEventListener("mouseup", handleHighlight);
 
 function handleHighlight() {
-    if (currentLabel == null || currentHighlighterColor == null) {
-        alert("Error: Please select a label and highlighter color first");
-        return;
-    }
-    let scrollPosition = textArea.scrollTop;
-    let range = getRangeOfSelectedText();
-
-    if (selectedInputRangeIsValid(range)) {
-        //build the annotation
-        let content = extractSelectedContent(range);
-        let notation = new Annotation(
-            range,
-            content,
-            currentHighlighterColor,
-            currentLabel
-        );
-
-        //then add this annotation to the current document
-        currentDocument.annotations.push(notation);
-
-        renderTextareaHighlights();
-        textArea.scrollTop = scrollPosition;
-    }
+  if (currentLabel == null) {
+    alert("Error: Please select a label and highlighter color first");
+    return;
+  }
+  let range = getRangeOfSelectedText();
+
+  if (selectedInputRangeIsValid(range)) {
+    //build the annotation
+    let content = extractSelectedContent(range);
+    let notation = new Annotation(
+      range,
+      content,
+      currentLabel
+    );
+
+    //then add this annotation to the current document
+    currentDocument.annotations.push(notation);
+
+    renderTextareaHighlights();
+  }
 }
 
 //returns the starting and ending position of the currently selected text
 getRangeOfSelectedText = function () {
-    let start = textArea.selectionStart;
-    let end = textArea.selectionEnd;
-    return {
-        "startPosition": start,
-        "endPosition": end
-    }
+  let start = textArea.selectionStart;
+  let end = textArea.selectionEnd;
+  return {
+    "startPosition": start,
+    "endPosition": end
+  }
 };
 
-function selectedInputRangeIsValid(range){
-  console.log(range.startPosition, range.endPosition);
-    return range.startPosition !== range.endPosition;
+function selectedInputRangeIsValid(range) {
+  console.log(range.startPosition, range.endPosition);              //remove when done
```

Comment:
We should keep the logs but enrich them with more descriptive data i.e. the function it was called in and what variables are being returned ("startposition = " + range.startPosition + "\n" + "endposition = " + range.endPosition)

Reply:
updated console logging to be more descriptive